acra/acra-in-depth/data-flow/_index.md

```diff
@@ -1,4 +1,113 @@
 ---
-title: Data flow(s)
+title: Data flow
 bookCollapseSection: true
 ---
+
+## Data flow
+
+### Acra components
+
+- [_AcraWriter_](/pages/documentation-acra/#acraconnector-and-acrawriter) — a client-side library that integrates into the app's workflow either through ORM or directly and provides the means for encryption of the sensitive data by generating AcraStructs.
```

Comment:
Is `/pages/` some kind of redirection? Because we only have `acra` and `themis` at the root of the repo.
Also, maybe `apps` would be better than `app's` since second one is generally short form of `app is`.

Reply:
nice catch, I missed this link